app/assets/stylesheets/comments.scss

```diff
@@ -345,6 +345,9 @@ a.header-link {
       code {
         word-wrap: break-word;
       }
+      .article-body-image-wrapper img{
```

Comment:
Would you mind adding some whitespace after `img`?

I know we don't have the best formatting in our CSS, but it's probably a good idea to work on it as we go forward. :smile:

Reply:
Sure! Also regarding the formatting thing, maybe the core maintainers can decide on a tool to enforce conventions too, like Prettier or Stylelint and take benefit of the ongoing Hacktoberfest with refactoring PRs like the Js, Jsx and ruby files have been.

Reply:
I will get back to you with a screenshot in some time, busy for a few days. I had to wipe out my Linux install again cause an update messed with windows and some of my volumes with important data went missing. >.<